src/SlackMessages.ts

```diff
@@ -82,8 +80,20 @@ export function atEveryone() {
     return "<!everyone>";
 }
 
+function hasItems(arr: any[]) {
+    return arr != null && arr.length > 0;
+}
+
 /** Renders JSON representation of slack message. */
 export function render(message: SlackMessage, pretty: boolean = false): string {
+    if (hasItems(message.attachments)) {
```

Comment:
JS won't allow it - I guess whatever code is being generated for this ends up using `length` property of the array so you end up with `Cannot read property 'length' of undefined` otherwise.